<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AticcoMainWIndow</class>
 <widget class="QMainWindow" name="AticcoMainWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>6</number>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="mainTab">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/graphics/png/graphics/png/components.png</normaloff>:/graphics/png/graphics/png/components.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="mainTabVerticalLayout" stretch="485,30,485">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="topHorizontalLayout">
            <item>
             <widget class="QWidget" name="CPUResourceConsumptionWIdget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_13"/>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="RAMResourceConsumptionWIdget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_12"/>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="middleHorizontalLayout" stretch="997,1003">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="bottomHorizontalLayout">
            <item>
             <widget class="QWidget" name="DiskResourceConsumptionWIdget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_10"/>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="NetworkResourceConsumptionWIdget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_11"/>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="hardwareTab">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/graphics/png/graphics/png/motherboard.png</normaloff>:/graphics/png/graphics/png/motherboard.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="hardwareMainVerticalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cpuTab">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/graphics/png/graphics/png/processor.png</normaloff>:/graphics/png/graphics/png/processor.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="cpuMainVerticalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ramTab">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/graphics/png/graphics/png/RAM.png</normaloff>:/graphics/png/graphics/png/RAM.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="ramMainVerticalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="diskTab">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/graphics/png/graphics/png/hardDisk.png</normaloff>:/graphics/png/graphics/png/hardDisk.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="diskMainVerticalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="networkTab">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/graphics/png/graphics/png/network.png</normaloff>:/graphics/png/graphics/png/network.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="networkMainVerticalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="usbTab">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/graphics/png/graphics/png/usb.png</normaloff>:/graphics/png/graphics/png/usb.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9" stretch="75,900">
        <item>
         <layout class="QHBoxLayout" name="usbTitleHorizontalLayout" stretch="900,100">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Report on the main USB bus status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshUsbViewButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="usbMainVerticalLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
